modules/payloads/singles/windows/loadlibrary.rb

```diff
@@ -11,7 +11,7 @@
 ###
 module MetasploitModule
 
-  CachedSize = 230
+  CachedSize = 202
```

Comment:
The loadlibrary payload sizes are smaller as the DLL was previously set to `external/source/byakugan/bin/XPSP2/detoured.dll` - now it's `data/vncdll.x86.dll`